plugin-java/src/test/java/appland/javaAgent/AppMapJavaAgentDownloadServiceTest.java

```diff
@@ -56,7 +56,7 @@ public void multipleDownloadRequests() throws IOException {
     public void lockFileGuard() throws IOException {
         var service = AppMapJavaAgentDownloadService.getInstance();
 
-        var jarAsset = GitHubRelease.getLatestRelease(new EmptyProgressIndicator(), "getappmap", "appmap-java")
+        var jarAsset = service.getDefaultRelease().getLatest(new EmptyProgressIndicator())
```

Comment:
With this change, is there still test coverage that the `getLatest()` is still working for GitHub?

Reply:
No, it doesn't. I've added another test: https://github.com/getappmap/appmap-intellij-plugin/pull/639/files#diff-59f897d536a484ce3373f5357e6a59093530b7aabbddf6a16de0ad9f792cb3d4R52